tracer/test/Datadog.Trace.ClrProfiler.Managed.Tests/DbScopeFactoryTests.cs

```diff
@@ -0,0 +1,127 @@
+// <copyright file="DbScopeFactoryTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Data;
+using Datadog.Trace.ClrProfiler.AutoInstrumentation.AdoNet;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.TestHelpers;
+using Xunit;
+using DbType = Datadog.Trace.ClrProfiler.AutoInstrumentation.AdoNet.DbType;
+
+namespace Datadog.Trace.ClrProfiler.Managed.Tests
+{
+    public class DbScopeFactoryTests
+    {
+        public static IEnumerable<object[]> GetDbCommands()
+        {
+            yield return new object[] { new System.Data.SqlClient.SqlCommand(),    nameof(IntegrationIds.SqlClient), DbType.SqlServer  };
+            yield return new object[] { new Microsoft.Data.SqlClient.SqlCommand(), nameof(IntegrationIds.SqlClient), DbType.SqlServer  };
+            yield return new object[] { new MySql.Data.MySqlClient.MySqlCommand(), nameof(IntegrationIds.MySql),     DbType.MySql      };
+            yield return new object[] { new Npgsql.NpgsqlCommand(),                nameof(IntegrationIds.Npgsql),    DbType.PostgreSql };
```

Comment:
Should we include _all_ DbCommands here? e.g. sqlite, oracle?

Reply:
I tried but had some difficulties with NuGet package compatibility, which is probably why we didn't have them before. I agree with you so I'll try again.

Reply:
fixed in 6d4067c6a71ff1181b1826fa7d8637dc84a3e210 and 5ba70c129fac6f89cd22a2abb344755b72fd765c by declaring our own mock `DbCommand`s with the correct type names